docs/Customs Declare Exports AutoComplete.js

```diff
@@ -126,10 +126,8 @@ function completePage() {
         selectFromAutoPredict(document.getElementById("country-container"), "GB");
         document.getElementsByClassName('button')[0].click()
     }
-    if (currentPageIs("/customs-declare-exports/declaration/destination-countries")) {
-        if(document.getElementById("countriesOfRouting-container")){
-            selectFromAutoPredict(document.getElementById("countriesOfRouting-container"), "GB")
-        }
+    if (currentPageIs("/customs-declare-exports/declaration/country-of-routing")) {
```

Comment:
Any change of some auto complete for the other new pages too? :)

Reply:
We should ideally have quick submit working no matter what page we go to

Reply:
ahh, yeah, magic tampermonkey, never remember about it. Thanks!

Reply:
Tampermonkey is set to skip this journey, destination countries are optional. Do we want to go through it or not?

It can be easily done in next PR, actual behaviour of script is correct

Reply:
I understand now what you mean, I'll prepare it in separate PR.